.github/workflows/lint.yml

```diff
@@ -23,6 +23,9 @@ jobs:
         with:
           node-version: '14'
           cache: 'yarn'
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
```

Comment:
Does it matter which go version is used here?

We are using go 1.19.4 in CI currently.  After 2.17 is released it will be upgraded to 1.19.5 or later according to Yi.

Reply:
Good point Matt. I thought we are on 1.18